app/routes/vet/cows-bvd-present-breeder.js

```diff
@@ -0,0 +1,47 @@
+const Joi = require('joi')
+const { vetVisitData: { vetBvdResult } } = require('../../session/keys')
+const session = require('../../session')
+const getYesNoInvestigateRadios = require('../helpers/yes-no-investigate-radios')
+
+const id = 'bvdRresult'
```

Comment:
Rather than using this id, the session key can be used. I think most of them are the same anyway. I've started to make this change in other routes. The benefit it has is to remove the need for the additional variable but mostly it means the session data is easier to reason about when you know it matches the ids of the inputs.

Reply:
Done